Signal/src/call/CallService.swift

```diff
@@ -142,7 +140,7 @@ protocol CallServiceObserver: class {
             updateIsVideoEnabled()
             updateLockTimerEnabling()
 
-            Logger.debug("\(self.TAG) .call setter: \(oldValue != nil) -> \(call != nil) \(String(describing: call))")
+            Logger.debug("\(self.TAG) .call setter: \(oldValue?.debugString as Optional) -> \(call?.debugString as Optional)")
```

Comment:
`SignalCall.debugString()` returns the triplet "remote phone number, local id, signaling id" which will make these logs more usable.